proof-market/application-developers/tutorials/signing-up.mdx

```diff
@@ -0,0 +1,39 @@
+import ApiSchema from '@theme/ApiSchema';
+
+# Signing up
+
+This tutorial showcases how authorization works in Proof Market and its API.
+
+:::info[Metamask]
+
+Authorization inside Proof Market requires a Metamask wallet.
+
+:::
+
+## Performing the initial request
+
+First, send a [**`GET` request to the `/user/metamask/message`**](../../api-reference#operation/MetamaskAuthMessage) endpoint to produce the following response.
+
+<ApiSchema pointer="#/components/schemas/MetamaskAuthMessage"/>
```

Comment:
Maybe we should expalin what each attribute is

Reply:
This can be generated automatically from the schema file, no need to add manual descriptions here